Source/CarthageKit/GitURL.swift

```diff
@@ -47,7 +47,16 @@ public struct GitURL {
 
 	/// The name of the repository, if it can be inferred from the URL.
 	public var name: String? {
-		let components = urlString.split(omittingEmptySubsequences: true) { $0 == "/" }
+        let absoluteURLString: String
```

Comment:
It looks like the new indentation is using spaces instead of tabs. Could you please reformat with tabs to match the existing indentation?

Reply:
Fixed in https://github.com/Carthage/Carthage/pull/2907/commits/e561595a6fdfe2bc47ca79866104c396575d8183.